src/main/java/ca/bc/gov/api/oracle/legacy/dto/ClientPublicFilterObjectVO.java

```diff
@@ -0,0 +1,26 @@
+package ca.bc.gov.api.oracle.legacy.dto;
+
+/**
+ * @author Maria Martinez, Government of BC
+ * @version 1.0.0
+ * @created 2022-11-03
+ */
+
+public class ClientPublicFilterObjectVO {
```

Comment:
This class can be changed to record

Reply:
Class is not being used so it's deleted.